### DISCUSSION TITLE
Streamlabs backend

### DISCUSSION THREAD

**Discussion by petitbonney** [letmaik/pyvirtualcam#85] [Q&A]
Hi guys, I'm interested in your repo. Is there a way to use the Streamlabs Virtual Camera, instead of original OBS one?
When I try to run the example code with Streamlabs VCam up, I get:
<pre>
RuntimeError: 'obs' backend: OBS Virtual Camera device not found! Did you install OBS?
'unitycapture' backend: No camera registered. Did you install any camera?
</pre>

**Comment by letmaik** [comment]
Is there any reason you can't use the OBS virtual camera?

Looking at the code of Streamlabs (https://github.com/stream-labs/slobs-virtual-cam), it seems that for Windows they use the implementation of the legacy OBS virtual cam plugin by catxfish (OBS has an integrated virtual camera now and doesn't require a plugin anymore), and for macOS they use the implementation from Webcamoid.

In pyvirtualcam, I don't think adding support for the legacy Windows OBS virtual cam plugin makes sense. It may make sense to support Webcamoid for macOS although there has been a thread on that already: https://github.com/letmaik/pyvirtualcam/issues/71.

If you can explain your use case a bit better then maybe that would help. A simple fix for you would be to just install OBS.
